nengo/spa/spa.py

```diff
@@ -80,153 +76,20 @@ def __init__(self):
 
     def __init__(self, label=None, seed=None, add_to_container=None,
                  vocabs=None):
-        super(SPA, self).__init__(label, seed, add_to_container)
+        warnings.warn(DeprecationWarning(
+            "nengo.spa.SPA is deprecated. Use nengo.spa.Module instead."))
+
+        if seed is not None:
+            rng = np.random.RandomState(seed)
+        else:
+            rng = None
         vocabs = [] if vocabs is None else vocabs
```

Comment:
Why does the SPA constructor behave differently than the standard Module constructor?  Does this mean there are situations where we can't just replace a SPA with a Module?